externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SparkSQLEngineListener.scala

```diff
@@ -18,6 +18,7 @@
 // Some object likes `JobFailed` is only accessible in org.apache.spark package
 package org.apache.spark.kyuubi
 
+import org.apache.spark.SparkException
```

Comment:
The PR changes touched code out scope of `test`, so remove the `[TEST]` in title and rename title to match the real changes.